src/core/libmaven/mavenparameters.cpp

```diff
@@ -33,14 +33,18 @@ MavenParameters::MavenParameters(string settingsPath):lastUsedSettingsPath(setti
         * Whenever we create an instance of this class, massCutoffType must be set for fragmentTolerance
         */
         fragmentTolerance = new MassCutoff();
-        fragmentTolerance->setMassCutoff(1000);
-        mzBinStep = 0.01;
-        rtStepSize = 20;
+        fragmentTolerance->setMassCutoffAndType(20, "ppm");
+        minFragMatchScore = 0;
+        minFragMatch = 3;
+        
+        
         /*
         * Whenever we create an instance of this class, massCutoffType must be set for massCutoffMerge
         */
         massCutoffMerge = new MassCutoff();
-        massCutoffMerge->setMassCutoff(30);
+        massCutoffMerge->setMassCutoffAndType(30, "ppm");
```

Comment:
This not being set was often the cause of some unexpected crashes. Good work setting it in the constructor itself.